models/clipper/ethereum/clipper_ethereum_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+        schema = 'clipper_ethereum_trades',
```

Comment:
please remove ```schema``` property here, the default from dbt_project yaml will take care of it. we don't want to override.